### DISCUSSION TITLE
Pretrained model

### DISCUSSION THREAD

**Discussion by QIQIVTrackingY** [milesial/Pytorch-UNet#282] [General]
Can you provide me a pretrained model? Thank you very much

**Comment by milesial** [comment]
If enough people want this, I could run a training on the Carvana dataset and share the weights. But anyone with a NVIDIA GPU could train the model on it in a few hours.

EDIT: [see below for the pretrained model](https://github.com/milesial/Pytorch-UNet/issues/189#issuecomment-666034521)

**Comment by VictorZoo** [comment]
Do we have any pre-trained models?

**Comment by milesial** [comment]
@VictorZoo Not yet, do you want one for the Carvana dataset?

**Comment by Nikronic** [comment]
Hi, as I used this repo a long time ago, I could remember there was a `MODEL.pth` file. So, I searched for it and here is the link:
https://github.com/milesial/Pytorch-UNet/blob/e2e46ce509382a45b1db4e1f639aeed568f6cb3e/MODEL.pth

**Comment by milesial** [comment]
Warning, the repo at that version was quite buggy, and this model is not compatible with the current version.

**Comment by milesial** [comment]
I will train a Carvana model at 10 reactions on the original comment.

**Comment by VictorZoo** [comment]
> I will train a Carvana model at 10 reactions on the original comment.

Okay! Thank you very much!

**Comment by buptlxy** [comment]
I want a pretrained model too! Thank you very much!

**Comment by VladimirBychkov** [comment]
Please add pretrained model! Thanks!

**Comment by msnh2012** [comment]
+1

**Comment by fitsir** [comment]
I want a pretrained model too! Thank you very much!

**Comment by ninanikitina** [comment]
Hi! I want the model too! Thank you!

**Comment by milesial** [comment]
Hello everyone, the Carvana model is available [in the releases](https://github.com/milesial/Pytorch-UNet/releases/tag/v1.0).
This was trained for 5 epochs, with scale=1 and bilinear=True. Let me know if you want one with transposed convs.

You can also use torch.hub to load it:

```
torch.hub.load('milesial/Pytorch-UNet', 'unet_carvana')
````

[More info about torch.hub](https://pytorch.org/docs/stable/hub.html?highlight=hub#module-torch.hub)
